.expeditor/config.yml

```diff
@@ -10,6 +10,9 @@ docker_images:
       build_args:
         - GEM_SOURCE: http://artifactory.chef.co/omnibus-gems-local
 
+habitat_packages:
+  - inspec
```

Comment:
I believe this needs/should be a key...you might want to optionally export a container to docker hub:

```yaml
habitat_packages:
  - inspec:
      export:
        - docker
```

This would export the current build and tag with a version (`MAJOR.MINOR.PATCH-YYYYMMDDHHMMSS`), channel (`unstable`). We can then "promote" the image (e.g. update channel tags) along with the omnibus packages via Expeditor artifact_actions.

Reply:
I was going off of the docs which make it seem like it can be a simple string as long as there were no options:

http://expeditor-docs.es.chef.io/configuration/habitat-packages/#habitat_packages

The InSpec docker image is built and consumed in a different way currently that I'd like to change in the future, but for now I need to keep them separate. Expeditor is at least handling the docker stuff for us, so that's cool.